crates/task-impls/src/request.rs

```diff
@@ -253,6 +254,8 @@ struct DelayedRequester<TYPES: NodeType, I: NodeImplementation<TYPES>> {
     pub network: Arc<I::Network>,
     /// Shared state to check if the data go populated
     state: Arc<RwLock<Consensus<TYPES>>>,
```

Comment:
Nit: not really related to the PR, but could this be called `consensus` rather than `state` for consistency with the other tasks?

Reply:
Yeah it should be I want to follow up on this pr with one that handles the VID stuff a little smarter

Reply:
Sounds good! reapproved